src/TConsole.cpp

```diff
@@ -1648,21 +1600,20 @@ void TConsole::setDisplayAttributes(const TChar::AttributeFlags attributes, cons
 void TConsole::setFgColor(int r, int g, int b)
 {
     setFgColor(QColor(r, g, b));
-    mUpperPane->forceUpdate();
```

Comment:
These cause an unneeded redraw as the method called on the previous line already does it.